File: CloudBridge-PM/src/CloudBridge/network/protocol/packet/LogPacket.php
<?php

namespace CloudBridge\network\protocol\packet;

class LogPacket extends Packet {

    public string $message = "";

    public function encode(): void {
        parent::encode();
        $this->put($this->message);
    }

    public function decode(): void {
        parent::decode();
        $this->message = $this->get();
    }

    public function getId(): int {
        return self::ID_LOG;
    }

    public static function create(string $message): self {
        $pk = new self;
        $pk->message = $message;
        return $pk;
    }
}

File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/LogPacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class LogPacket extends Packet {

    public Object message = "";

    @Override
    public void encode() {
        super.encode();
        put(message);
    }

    @Override
    public void decode() {
        super.decode();
        message = get();
    }

    @Override
    public Integer getId() {
        return ID_LOG;
    }

    public static LogPacket create(String message) {
        LogPacket pk = new LogPacket();
        pk.message = message;
        return pk;
    }
}
